### PR TITLE
[MM-47634] Changed slash command description text

### PR DESCRIPTION
**Pull request by WilliamLongKing**
#### Summary
Changed slash command description help text

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-47634

#### Screenshots
image
The description text has been changed from "Describe your incoming webhook." to "Describe your slash command."

#### Release Note
```release-note
Fixed the slash command description help text
```

### CODE REVIEW

File: i18n/en_AU.json [hunk 1]
```diff
@@ -84,7 +84,7 @@
   "add_command.autocompleteHint.placeholder": "Example: [Patient Name]",
   "add_command.cancel": "Cancel",
   "add_command.description": "Description",
-  "add_command.description.help": "Describe your incoming webhook.",
```
Comment (mickmister) [inline i18n/en_AU.json:87]: I think this is generated by our translation server, so this shouldn't be updated in this PR

**Review by mickmister — APPROVED** [review]
LGTM other than I think we should remove the `en_AU.json` changes. Nice work @WilliamLongKing :+1:

**Comment by mickmister** [comment]
@DHaussermann I'm thinking we can defer QA review on this one. Please let me know what you think :+1:

**Review by hanzei — APPROVED** [review]
LGTM :+1:

**Comment by hanzei** [comment]
@WilliamLongKing Could you please change the status in JIRA to submitted?

**Review by DHaussermann — APPROVED** [review]
I agree with @mickmister.

This can be merged. I will explicitly verify this post-merge as part of release testing.

Thanks @WilliamLongKing  for making the change!

**Comment by DHaussermann** [comment]
@WilliamLongKing sorry I was delayed and getting to this. Can you look at these merge conflicts?

**Comment by WilliamLongKing** [comment]
@DHaussermann what conflicts might you be referring to? I'm seeing that there are no conflicts with the base branch.

**Comment by hanzei** [comment]
I also don't see any merge conflicts.

**Comment by DHaussermann** [comment]
I got this response and assumed there were merge conflicts.
image

If not please merge and I will cover this change as part of release testing 👍

**Comment by WilliamLongKing** [comment]
will merge

**Pull request merged**